python/cuspatial/cuspatial/demos/traj_test_soa_locust.py

```diff
@@ -30,17 +30,11 @@
 y, m, d, hh, mm, ss, wd, yd, ms, pid = tools.get_ts_struct(ts_0)
 
 pnt_x, pnt_y = gis.cpp_lonlat2coord(cam_lon, cam_lat, pnt_lon, pnt_lat)
-num_traj, tid, len, pos = traj.cpp_coord2traj(pnt_x, pnt_y, id, ts)
-
+num_traj, trajectories = traj.cpp_derive_trajectories(pnt_x, pnt_y, id, ts)
```

Comment:
Why is this demo using the bindings directly rather than the Python API?  e.g. it should work like cuDF. Maybe I misunderstood your previous PRs, but there should be a `cuspatial.derive_trajectories` so users don't have to call into `cuspatial.bindings`.